File: Ticket_2_Unit_Test_Examples/NotamUtilsTest.java
import java.util.*;

public class NotamUtilsTest {

    public static void main(String[] args) {

        var notams = NotamUtils.fetchNotams();
        assert notams.size() == 2;
        assert notams.get(0).id.equals("A1");
        assert notams.get(1).urgency.equals("high");

        var raw = new HashMap<String, String>();
        raw.put("id", "C3");
        raw.put("type", "Runway Maintenance");
        raw.put("location", "KJFK");

        var processed = NotamUtils.processNotam(raw);
        assert processed.urgency.equals("normal");

        System.out.println("Tests passed.");
    }
}

File: Ticket_2_Unit_Test_Examples/NotamUtilsTest.cpp
#include <iostream>
#include <cassert>
#include <unordered_map>

// Include the implementation file directly
#include "NotamUtils.cpp"

int main() {
    auto notams = NotamUtils::fetchNotams();
    assert(notams.size() == 2);
    assert(notams[0].id == "A1");
    assert(notams[1].urgency == "high");

    std::unordered_map<std::string, std::string> raw;
    raw["id"] = "C3";
    raw["type"] = "Runway Maintenance";
    raw["location"] = "KJFK";

    auto processed = NotamUtils::processNotam(raw);
    assert(processed.urgency == "normal");

    std::cout << "Tests passed." << std::endl;
    return 0;
}
